docs/deploy/docker.md

```diff
@@ -51,6 +51,19 @@ like:
 docker run --device /dev/kfd --device /dev/dri/renderD128 --device /dev/dri/renderD130 <image>
 ```
 
+### Additional Options
+
+The performance of an application can vary depending on the assignment of GPUs and CPUs to the task. Typically, numactl is installed as part of many 
+HPC applications to provide GPU/CPU mappings. This Docker runtime option supports memory mapping and can improve performance
+
+```--security-opt seccomp=unconfined```
+  
+This option is recommened for Docker Containers running HPC applications.
```

Comment:
Why is it recommended for HPC applications? It would be nice to know as this is impacts the end users security.

Reply:
Perhaps I am missing a ":"  

Would this be clearer -   This Docker runtime option supports memory mapping and can improve performance:

The line should describe the --security-opt option and explains why (performance) it is recommended. Please make any change needed to clarify that the security option is recommended because it allows for memory mapping and (therefore) improved performance.

Reply:
Now it is getting more interesting. I would like to link an article on our memory model and why it needs the --security-opt. Something for the future.

Reply:
Disabling seccomp filtering is a bad recommendation to make, it negates most security benefits docker gives you.

It would be much better to enable the syscall on top of docker defaults that is needed for better performance.

I couldn't find any mention of memory mapping that would be blocked (`mmap` and `mmap2` are allowed) by the default config. `mbind` is blocked though by default, which is for manipulating NUMA policies, maybe this is what you meant?
For this enabling the `CAP_SYS_NICE` capability is sufficient, but I would still mention where this might be needed (e.g. in NUMA aware applications that use the syscall or for using `numactl` in the container) instead of a blanket recommendation.

Reply:
I understand that the option is important to add for performance. However, let me do a little due diligence to see if there is an alternate secure mechanism to get the performance improvement.